paddle/fluid/operators/controlflow/fetch_op.cc

```diff
@@ -20,6 +20,32 @@ limitations under the License. */
 
 namespace paddle {
 namespace operators {
+void TransDataLayout(const framework::LoDTensor &src_var,
+                     std::string fetch_var_name,
```

Comment:
Do not pass value of object in C++, it will be slow. Please use "const std::string&"